### PR TITLE
SW-18: implement default maxLength for TEXT

### PR DESCRIPTION
**Pull request by YuraKovalenko**
implemented default maxLength for type TEXT

### CODE REVIEW

File: schema-wizard-core/src/main/java/org/schemawizard/core/property/service/impl/PropertyParserImpl.java [hunk 1]
```diff
@@ -45,6 +46,12 @@ public boolean parseBooleanValue(String value) {
         return Boolean.parseBoolean(stringValue);
     }
 
+    @Override
+    public Integer parseLongValue(String value) {
+        String stringValue = parseStringValue(value);
+        return Integer.parseInt(stringValue);
+    }
+
```
Comment (AndriyBosik) [inline schema-wizard-core/src/main/java/org/schemawizard/core/property/service/impl/PropertyParserImpl.java:54]: also, `Integer.parseInt()` throws an error when null is passed, which will lead to an exception threw when `parseStringValue()` returns null

File: schema-wizard-core/src/main/java/org/schemawizard/core/model/defaults/Text.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        public Text build() {
+            return new Text(defaultLength);
+        }
+    }
+}
+
+
+//public Text build() {
+//    return new Text(defaultLength);
+//}
```
[the remark this answers is not in the document]
Comment (YuraKovalenko) [reply schema-wizard-core/src/main/java/org/schemawizard/core/model/defaults/Text.java]: fixed

File: schema-wizard-core/src/main/java/org/schemawizard/core/property/service/impl/PropertyParserImpl.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (YuraKovalenko) [reply schema-wizard-core/src/main/java/org/schemawizard/core/property/service/impl/PropertyParserImpl.java]: fixed

Comment (YuraKovalenko) [reply schema-wizard-core/src/main/java/org/schemawizard/core/property/service/impl/PropertyParserImpl.java]: fixed

File: schema-wizard-core/src/main/java/org/schemawizard/core/property/service/impl/ConfigurationPropertiesServiceImpl.java [hunk 3]
```diff
@@ -1,10 +1,14 @@
 package org.schemawizard.core.property.service.impl;
 
-import org.schemawizard.core.exception.InvalidConfigurationPropertiesLocation;
+import java.io.InputStream;
+import java.net.URL;
```
Comment (AndriyBosik) [inline schema-wizard-core/src/main/java/org/schemawizard/core/property/service/impl/ConfigurationPropertiesServiceImpl.java:4]: do we need these imports here?

**Review by AndriyBosik — APPROVED** [review]

File: schema-wizard-core/src/main/java/org/schemawizard/core/property/service/impl/ConfigurationPropertiesServiceImpl.java (hunk 3 above)
Comment (YuraKovalenko) [reply schema-wizard-core/src/main/java/org/schemawizard/core/property/service/impl/ConfigurationPropertiesServiceImpl.java:4]: yes, because we use it

**Pull request merged**